clusters/cik8s.yaml

```diff
@@ -13,7 +13,17 @@ repositories:
   - name: jenkins-infra
     url: https://jenkins-infra.github.io/helm-charts
 releases:
+  - name: docker-registry-secrets
+    namespace: default
```

Comment:
I wouldn't put it in the "default" namespace, maybe "docker-registry" here?

Reply:
agreed, but @dduportal said to go by step ;) ... 
still a WIP

Reply:
We discovered that specifying a nemspace for the helmfile release is required (despite what the doc. says) to allow referencing from other release (with the `needs` keyword).

Since this helm chart does not create any resource in the aformentioned namespace (it creates in the value-specified namespaceS), I proposed to use "defaukt" here, as a "no-op" value.

Stephane just added a comment in the code to explain this. WDYT?

Reply:
I dunno about the "default" namespace, which is frenquently used for tests or install without being cautious, and subject to deleting. Not a big problem per se, but I'd put it in "jenkins-agents", WDYT?